extensions/ql-vscode/src/view/variant-analysis/__tests__/VariantAnalysisActions.spec.tsx

```diff
@@ -45,6 +45,24 @@ describe(VariantAnalysisActions.name, () => {
     expect(onStopQueryClick).toHaveBeenCalledTimes(1);
   });
 
+  it("renders the stopping query disabled button when canceling", async () => {
```

Comment:
Do we need more tests?

For example:
- the status is set to cancel when the button is clicked
- the status is set back to in-progress if the request to cancel fails
- the monitoring job retains the canceling status